docs/en/interfaces/cli.md

```diff
@@ -65,6 +65,31 @@ You can cancel a long query by pressing Ctrl+C. However, you will still need to
 
 The command-line client allows passing external data (external temporary tables) for querying. For more information, see the section "External data for query processing".
 
+### Formatted Queries {#cli-formatted-queries}
+
+You can create a format string for the query, and provide arguments for the format string with the parameters of the client app. For example:
+
+```bash
+clickhouse-client --param_parName="[1, 2]"  -q "SELECT * FROM table WHERE a = {parName:Array(UInt16)}"
+```
+
+#### Syntax of a Format String {#cli-formatted-queries-syntax}
+
+Format a query by the standard method. Values that you want to put into the query from the app parameters place in braces and format as follows:
+
+```
+{<name>:<structure>}
```

Comment:
why structure? it's just data type

Reply:
If you look at the examples you will see that it describes data structure, For example `Tuple(UInt8, Tuple(String, UInt8))`

Reply:
It's still just a complex data type.